core-guides/core-data-curators.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Welcome, Data Curators
+---
+
+As mentioned in the introductory page, one key issue is that the problem with open data is not a matter of *creation*, rather *curation*. This means the role of a Data Curator is to take data from known (quality-proven and reliable) sources and 
+transform that into data packages.
+
+This guide will introduce you to key routines we usually do, before publishing any data package.
+
+# Getting data
+
+The first step is, naturally, finding data to work with. There are two common ways you can do this:
+
+1. Follow the [GitHub's issue tracker](https://github.com/datasets/registry/issue) and look for a package with 3-star priority, or
+
+2. Get your own data and start preparing it.
+
+Which one you should go to first is really up to you. The difference, however, is that some are more needed than others and 
+you could use that knowledge and skill to tackle on some of the project's priorities.
```

Comment:
I think it doesn't change the outlook but I also did that because it is easy to scroll through text in vim. I can fix it though.